README.md

```diff
@@ -37,26 +39,31 @@ GRAFANA_PASSWORD=admin" > .env_hobby
 ```
 
 3. Launch services:
+
 ```bash
 docker-compose --env-file .env_hobby -f docker-compose.yml up -d
 ```
 
 4. Issue one-time invite token:
+
 ```bash
 docker-compose --env-file .env_hobby -f docker-compose.yml run engine python manage.py issue_invite_for_the_frontend --override
 ```
 
+**Note**: if you remove the plugin configuration and reconfigure them, you will need to generate a new one-time invite token for your new configuration.
```

Comment:
This line is the only actual change in this file (rest is markdown autoformatter changes)